README.md

```diff
@@ -0,0 +1,138 @@
+
+Now you have a bucket name you will create the configuration for your
+stack. Inside the `stacks` directory you will find a pair of files
+for each stack, a `.backend` and a `.tfvars`. Make a copy of an existing
+pair and change the values to suit your new name. The `bucket`
+and `remote_state_bucket` settings in these files must match the bucket you
+created above.
+
+You should also ensure that the AWS account you are creating your environment in has an SSH key pair
+set up called `ecs-monitoring-ssh-test`. You should do this manually using the AWS web console. You
+will need to download the private key for this key pair when you create it if you wish to SSH in to
+the ECS node.
+
+### Creating your stack
+
+Once you've created your stack configurations, and added the
+correct bucket name, you can create the stack:
+
+    cd terraform/projects/infra-networking
+
+    $ terraform init -backend-config=../../../stacks/staging.backend
+
+    $ terraform plan -var-file=../../../stacks/staging.tfvars
+
+    $ terraform apply -var-file=../../../stacks/staging.tfvars
+
+    cd ../infra-security-groups
+
+    # terraform commands from above
+
+    cd ../app-ecs-nodes
+
+    # terraform commands from above
+
+    cd ../app-ecs-albs
+
+    # terraform commands from above
+
+    cd ../app-ecs-services
+
+    # terraform commands from above
+
+## Development process
+
+If you want to make a change to our Prometheus infrastructure you should:
+
+- Create a new branch
+- Create a new stack for testing purposes in the team sandbox AWS account by following the above set up instructions
+- Once you are happy with your code, put in a pull request and get it reviewed by another team member
+- Once your PR is merged, manually deploy to the staging stack in the staging AWS account using Terraform
+- If staging is fine then manually deploy to the production stack in the production AWS account using Terraform
+
+
+## Creating documentation
+
+The projects in this repo use the [terraform-docs](https://github.com/segmentio/terraform-docs)
+to generate the per project documentation.
+
+You can install `terraform-docs` by running:
+
+    brew install terraform-docs
+
+When adding adding or changing terraform projects you should run `terraform-docs` 
+in the project directory and add that to your commit, for example:
+
+    cd terraform/projects/app-ecs-albs
+    terraform-docs markdown . > README.md
+
+## AWS Vault tips
+
+Every time you execute AWS vault it may ask for the credentials to access the keychain.
+
+To avoid this, we can follow the next steps (in OS X):
+
+1. Open the 'Keychain Access' utility.
+2. In the menu select "File > Add Keychain..."
+3. Select the "aws-vault.keychain". It can be found in `~/Library/Keychains/`.
+4. Once it is added, right click in it (it shows in the left hand side under 'Keychains') and select `Change Settings for Keychain aws-vault`.
+5. Uncheck `Lock after X minutes of inactivity` and `Look when sleeping`
```

Comment:
I got a bit worried when reading this.  If I uncheck these options, when will my keychain ever lock again?

(I don't know a lot about OS X Keychain)

Reply:
after a bit of digging I think this advice is wrong.

There's two different things going on:
 - access for aws-vault app to the keychain
 - whether the keychain is locked or unlocked

By default, you have to type the password each time you want the app to access the keychain, whether the keychain itself is locked or not.  To prevent this, you click "always allow" instead of "allow" in the pop up dialog.

Also by default, the keychain locks after your computer sleeps, or after 5 minutes of inactivity.  After that, any use of the keychain requires the password again.

I think we should recommend users click "always allow" to allow aws-vault to access the keychain without prompting every time, but do not change the "lock after X minutes" or "lock when sleeping" settings.